src/elements/Button/Button.story.tsx

```diff
@@ -157,3 +223,56 @@ export const CustomTheme = () => {
     </ThemeProvider>
   );
 };
+
+export const Gradients = () => {
```

Comment:
This is confusing to me - as a user I would kind of expect this to be a variant.

Reply:
We already have Variants